airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/tests/test_incremental.py

```diff
@@ -51,7 +51,7 @@ def future_state_fixture(future_state_path) -> Path:
 
 @pytest.fixture(name="cursor_paths")
 def cursor_paths_fixture(inputs, configured_catalog_for_incremental) -> Mapping[str, Any]:
-    cursor_paths = getattr(inputs, "cursor_paths")
+    cursor_paths = getattr(inputs, "cursor_paths") or {}
```

Comment:
what is the impact of this change?

Reply:
cursor_paths = getattr(inputs, "cursor_paths") returns None if cursor_paths are commented in yaml, and this case fail few lines below on cursor_paths.get